src/modules/Crowbar.rb

```diff
@@ -195,7 +202,15 @@ def Read
       @conduit_map = network["conduit_map"] || []
 
       @template_crowbar = json2hash(@crowbar_file)
-      @users = @template_crowbar["attributes"]["crowbar"]["users"] rescue {}
+
+      begin
+        crowbarrc = IniFile.load("/etc/crowbarrc")
+        @admin_user = crowbarrc["default"]["username"]
+        @admin_password = crowbarrc["default"]["password"]
+      rescue IniFile::Error
+        Report.Error(Message.CannotOpenFile(@crowbarrc_file))
```

Comment:
I really recommend to log error message here. Really useful in bug reports.

Reply:
Doesn't Report module log the error?

Reply:
`Report.Error` itself does the logging, the disadvantage is that uses the translated message so it might be unreadable when the user uses some exotic language. On the other hand from the file and line number you can find the original place where it was reported. So it's up to you...

Reply:
@jsuchome as lada writes, generic message that file cannot be opened is not much helpful when you get it from customer. so using 
```
rescue IniFile::Error => e
  Yast.y2error "Failed to open file #{@crowbarrc_file}: with #{e.message}"
```